cmd/web.go

```diff
@@ -236,9 +242,12 @@ func runWeb(ctx *cli.Context) error {
 		m.Post("/email/delete", user.DeleteEmail)
 		m.Get("/password", user.SettingsPassword)
 		m.Post("/password", bindIgnErr(auth.ChangePasswordForm{}), user.SettingsPasswordPost)
-		m.Get("/openid", user.SettingsOpenID)
-		m.Post("/openid", bindIgnErr(auth.AddOpenIDForm{}), user.SettingsOpenIDPost)
-		m.Post("/openid/delete", user.DeleteOpenID)
+		// TODO: only if OpenID is enabled ?
+		m.Group("/openid", func() {
```

Comment:
You mean I should change the routes ? Because now we have `/login/openid` and `/openid/etc..`